delete_project.yml

```diff
@@ -15,9 +15,18 @@
 
     - import_tasks: tasks/set_vars.yml
 
+    - name: Check if project exists
```

Comment:
Do you think this task is really needed ? It fails now if the project doesn't exist ?

Reply:
I should re-test it but yes it failed during my first tests :cry: